modules/local/microfinder_filter/main.nf

```diff
@@ -0,0 +1,48 @@
+process MICROFINDER_FILTER {
+    tag "$meta.id"
+    label "process_single"
+
+    conda "conda-forge::coreutils=9.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+    'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
+    'docker.io/ubuntu:20.04' }"
+
+    input:
+    tuple val(meta), path(input_file)
+    tuple val(meta), path(input_assembly)
+    val (output_prefix)
```

Comment:
Try to use the default `prefix` defined below.
I've tried to trace back `output_prefix`. If I get it correctly, it's `meta.id` unless someone defines `params.output_prefix`. Then something like this could work
```groovy title="conf/modules.config"
process {
    withName: MICROFINDER_FILTER {
        ext.prefix = params.output_prefix
    }
}
```
and no need to have an `output_prefix` channel